server.js

```diff
@@ -1,12 +1,16 @@
 /**
  * Module dependencies.
  */
+
+// Require 'dotenv' config
+require('dotenv').config();
+
+
 var express = require('express'),
     fs = require('fs'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
-require('dotenv').config();
```

Comment:
But in the current position it's in the same block with all the `require` statements existing.